cellxgene_schema_cli/cellxgene_schema/migrate.py

```diff
@@ -365,98 +67,32 @@ def migrate(input_file, output_file, collection_id, dataset_id):
-    # https://github.com/chanzuckerberg/single-cell-curation/blob/43f891005fb9439dbbb747fa0df8f0435ebf3f7c/cellxgene_schema_cli/cellxgene_schema/validate.py#L761-L762
-    for key, value in list(dataset.uns.items()):
-        if any(
-            isinstance(value, sparse_class)
-            for sparse_class in (scipy.sparse.csr_matrix, scipy.sparse.csc_matrix, scipy.sparse.coo_matrix)
-        ):
-            if value.nnz == 0:  # number non-zero
-                del dataset.uns[key]
-        elif value is not None and not isinstance(value, (np.bool_, bool)) and len(value) == 0:
-            del dataset.uns[key]
-
-    column_changes = []
-    for key in list(dataset.obsm.keys()):
-        delete_key = False
-        new_key = key
-        if " " in new_key:
-            new_key = new_key.replace(" ", "_")
-            delete_key = True
-        if new_key.startswith("_"):
-            new_key = new_key.strip("_")
-            delete_key = True
-        if "(" in new_key or ")" in new_key:
-            new_key = new_key.replace("(", "").replace(")", "")
-            delete_key = True
-        if delete_key:
-            dataset.obsm[new_key] = dataset.obsm[key]
-            if (default_embedding := dataset.uns.get("default_embedding")) and default_embedding == key:
-                dataset.uns["default_embedding"] = new_key
-            column_changes.append(key)
-            del dataset.obsm[key]
-
-    if "X_.umap_MinDist_0.2_N_Neighbors_15" in dataset.obsm:
-        # Applies to dataset ids 63bb6359-3945-4658-92eb-3072419953e4 and 9b188f26-c8e1-4a78-af15-622a35a371fc
-        dataset.obsm["X_umap_MinDist_0.2_N_Neighbors_15"] = dataset.obsm["X_.umap_MinDist_0.2_N_Neighbors_15"]
-        del dataset.obsm["X_.umap_MinDist_0.2_N_Neighbors_15"]
+    # https://github.com/chanzuckerberg/single-cell-curation/issues/825
+    if collection_id == "0aab20b3-c30c-4606-bd2e-d20dae739c45":
+        utils.replace_ontology_term(dataset.obs, "disease", {"MONDO:0060782": "MONDO:0100542"})
 
-    if collection_id == "91c8e321-566f-4f9d-b89e-3a164be654d5":
-        utils.map_ontology_term(
-            dataset.obs,
-            "cell_type",
-            "author_cell_type",
-            {
-                "L4-5IT_RORB_TSHZ2": "CL:4030062",
-                "L2-4IT_CUX2": "CL:4030059",
-                "L4-5IT_RORB_LRRK1": "CL:4030062",
-                "L4-5IT_RORB_ARHGAP15": "CL:4030062",
-                "L6IT_THEMIS_LINC00343": "CL:4030065",
-                "L6IT_THEMIS_CUX1": "CL:4030065",
-                "L3-5IT_RORB_PLCH1": "CL:4030061",
-            },
-        )
+    if "X_spatial" in dataset.obsm_keys():
+        del dataset.obsm["X_spatial"]
 
-    if collection_id == "e1fa9900-3fc9-4b57-9dce-c95724c88716":
-        utils.replace_ontology_term(dataset.obs, "cell_type", {"CL:4023040": "CL:4030059"})
+    if dataset.uns.get("default_embedding") == "X_spatial":
+        dataset.uns["default_embedding"] = "spatial"
```

Comment:
any edge cases around this to account for? any scenario where an existing dataset in the corpus has X_spatial but not spatial yet?

Reply:
There shouldn't be.
And I think this can be confirmed via API - pretty sure all embeddings (X_ or not) are returned

Reply:
@jahilton ?

Reply:
I'll take a look at the API, thanks